protocol/x/prices/keeper/currency_pair_id_cache.go

```diff
@@ -0,0 +1,44 @@
+package keeper
+
+// CurrencyPairIDCache handles the caching logic of currency-pairs to their corresponding IDs
+type CurrencyPairIDCache struct {
```

Comment:
can you add a comment that this cache is lazily updated and does not necessarily contain all CPs?